src/uct/api/uct.h

```diff
@@ -1807,10 +1812,11 @@ ucs_status_t uct_ep_create(const uct_ep_params_t *params, uct_ep_h *ep_p);
 
 /**
  * @ingroup UCT_CLIENT_SERVER
- * @brief Initiate a synchronized disconnection of an endpoint connected to a
+ * @brief Initiate a disconnection of an endpoint connected to a
  *        sockaddr by a connection manager @ref uct_cm_h.
  *
- * This routine will disconnect the given endpoint from its remote peer.
+ * This non-blocking routine will disconnect the given endpoint from its remote
+ * peer.
```

Comment:
how we can "wait" on the non-blocking request to complete ?

Reply:
maybe ` This non-blocking routine will send a disconnect notification on the endpoint, so that @ref uct_ep_disconnect_cb_t will be called on the remote peer`

Reply:
But when/how the local/initiator knows that the request was completed ?

Reply:
@alinask - I'm still missing if there is a way to `quite` on the disconnect request ?

Reply:
@shamisp there is no notion of "local completion of a disconnect request". it's just sent and that's it (like am_bcopy/am_short for example).

Reply:
@yosefe for other two we have flush semantics that essentially waits for ack.

Reply:
so do you think we should say that uct flush also makes sure the disconnect arrived to remote side?
for these two we still dont have local completion..

Reply:
Sorry, I didn't mean local completion. I had in mind remote completion. Yes, I think we have to update the flush semantics to cover the disconnect request.